### PR TITLE
fix(mqtt): fixed how mqtt library is imported

### PR DESCRIPTION
**Pull request by csarnataro**
Because of some issues with some bundlers (namely, Parcel 2) we have updated the way mqtt.connect is imported in Connection.ts

Minor updates to index.ts (eslint)

Added a changelog section in readme file

### CODE REVIEW

File: src/connection/Connection.ts [hunk 1]
```diff
@@ -106,7 +111,8 @@ export class Connection implements IConnection {
       else valueToSend = value;
     });
 
-    if (valueToSend !== {}) messages.push({ topic, propertyName: current, value: valueToSend });
+    // the condition `if (valueToSend !== {}) ` has been removed bc it always evaluates to true
```
Comment (fabioimpe) [inline src/connection/Connection.ts:114]: **thought**: maybe here is worth understanding if is better to remove the `if` or leaving the `if` fixing the condition

Comment (csarnataro) [reply src/connection/Connection.ts:114]: Ok, it makes sense, I'm going to see if I can understand the original intention

File: src/connection/Connection.ts [hunk 2]
```diff
@@ -106,7 +111,11 @@ export class Connection implements IConnection {
       else valueToSend = value;
     });
 
-    if (valueToSend !== {}) messages.push({ topic, propertyName: current, value: valueToSend });
+    // If the message is an object, and it's empty, it makes no sense to send it
+    // All other messages (e.g. non-empty objects or primitives) must be sent
+    if (!(typeof valueToSend === 'object' && Object.keys(valueToSend).length == 0)) {
```
Comment (csarnataro) [inline src/connection/Connection.ts:116]: @fabioimpe do you think it's ok now, with an explicit check on the object and its length?
I think now is more explicit that:
- if it's an empty object, skip sending the message
- if it's a non-empty object, or any other value (primitive) then send it.

I thing it's rather an edge-case, which probably happens if the `properties` array is empty (so `forEach` statement is never executed) or `value` at line 111 is an empty object.

Comment (fabioimpe) [reply src/connection/Connection.ts:116]: Yes I think now is cleaner.
Maybe you can think about extracting the condition to be more readable, but this seems the right way 👍

**Pull request merged**